### ISSUE TITLE
Improper argument alignment for case classes with body. 

### ISSUE THREAD

**Issue by imarios** [sbt/sbt-scalariform#50]
Using the latest (?) version (`"org.scalariform" % "sbt-scalariform" % "1.6.0"`) I found a corner case where case classes arguments are (from what I believe) improperly aligned. 

Better to show this with an example:

```scala
case class RandomClassName(asdasdsd: Long, 
                                asdasdsadsadsadsads: String, 
                                asdasdsadas: Boolean
) {
  def xxx = 1
  def yyy = 2
}
```

The formatted code looks like this: 

```scala
case class RandomClassName(
  asdasdsd: Long,
    asdasdsadsadsadsads: String,
    asdasdsadas: Boolean
) {
  def xxx = 1
  def yyy = 2
}
```

The first argument is aligned with 2 space, the remaining at 4. This does not happen when the case classes don't have a body:

```scala
case class RandomClassName(asdasdsd: Long,
                                asdasdsadsadsadsads: String,
                                asdasdsadas: Boolean
)
```

Get's formatted as expected. 

```scala
case class RandomClassName(
  asdasdsd: Long,
  asdasdsadsadsadsads: String,
  asdasdsadas: Boolean
)
```

**Comment by imarios** [comment]
Could be a scalariform issue and not the plugin. Actually that seems to be more likely. Let me know if you believe this is case and I can open an Issue for them.

**Comment by fommil** [comment]
@imarios nobody is maintaining scalariform, if you'd like this fixed you'd be best writing up a test and then having a go at fixing it.

**Comment by imarios** [comment]
@fommil thank you Sam. I will look into this.

**Comment by godenji** [comment]
Should be fixed in latest version of sbt-scalariform (1.7.0) which depends on scalariform 0.2.1.
